manager.py

```diff
@@ -280,11 +282,13 @@ def run(self):
                         time.time() - self.interval)
 
                 self.handle_cpm(this_start, this_end)
+                self.extract_git_tag()
                 if self.quit_after_interval:
                     self.vprint(1, 'Reboot: taking down Manager')
                     self.stop()
                     self.takedown()
                     os.system('./git-pull-reboot.sh')
```

Comment:
Add `self.branch` as an argument here. I'll work on `git-pull-reboot.sh` and confirm, but for now let's say the branch comes directly after the script name: `./git-pull-reboot.sh master` for example.